src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -104,6 +104,14 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web config update', 'java_container_version', help="The version of the java container, e.g., '8.0.23' for Tomcat")
 register_cli_argument('appservice web config update', 'app_command_line', options_list=('--startup-file',), help="The startup file for linux hosted web apps, e.g. 'process.json' for Node.js web")
 
+register_cli_argument('appservice web config ssl bind', 'certificate_thumbprint', help='The ssl cert thumbprint')
+register_cli_argument('appservice web config ssl bind', 'ssl_type', help='The ssl cert type, e.g., SNI or IP')
```

Comment:
This way, values outside of that list would not be allowed and will be rejected when command parsing happens.